shell/packages/sandstorm-db/db.js

```diff
@@ -409,6 +411,8 @@ ApiTokens = new Mongo.Collection("apiTokens");
 //              `MembraneRequirement` structure defined in `supervisor.capnp`.
 //   hasApiHost: If true, there is an entry in ApiHosts for this token, which will need to be
 //              cleaned up when the token is.
+//   ownerSeenAllActivity: True if the owner has viewed the grain since the last activity event
```

Comment:
I'm not sure what happened here. `ownerSeenAllActivity` was supposed to be a field on Grains, but somehow I falsely documented a field called `ownerUnread` there and then documented the correct field on the wrong collection.